src/components/ui/textarea.tsx

```diff
@@ -0,0 +1,24 @@
+import { forwardRef } from 'react';
+
+import { cn } from '@/lib/utils';
+
+export interface TextareaProps
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ className, ...props }, ref) => {
+    return (
+      <textarea
+        className={cn(
+          'flex h-[283px] w-full rounded-xl border border-input border-slate-400 bg-white px-3 p-px pb-[18px] text-lg ring-offset-background placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
+          className,
```

Comment:
<img width="409" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/58718006/3eec8a3a-b7db-4522-942a-0f34a867cad7">

could you check the margin of the text box?